apps/contrib/tasks.py

```diff
@@ -154,3 +154,27 @@ def kill_all_long_running_report_generations():
     ).update(status=ReportGeneration.REPORT_GENERATION_STATUS.KILLED)
 
     logger.info(f'Updated REPORT GENERATION to killed:\n{progress=}')
+
+
+@celery_app.task
+def generate_idus_dump_file():
+    import json
+    from apps.entry.models import ExternalApiDump
+    from apps.entry.serializers import FigureReadOnlySerializer
+    from apps.entry.views import get_idu_data
+    from utils.common import get_temp_file
+
+    external_api_dump, created = ExternalApiDump.objects.get_or_create(
+        api_type=ExternalApiDump.ExternalApiType.IDUS.value,
+    )
+    try:
+        serializer = FigureReadOnlySerializer(get_idu_data(), many=True)
+        with get_temp_file(mode="w+") as tmp:
+            json.dump(serializer.data, tmp)
+            external_api_dump.dump_file.save('idus_dump.json', File(tmp))
+        external_api_dump.status = ExternalApiDump.Status.COMPLETED.value
```

Comment:
We don't need to use `.value` in this cases.
```suggestion
        external_api_dump.status = ExternalApiDump.Status.COMPLETED
```